app/controllers/accounts_controller.rb

```diff
@@ -78,6 +78,14 @@ def destroy
 
   private
 
+    def decide_layout
+      if can? :read, :admin_dashboard
```

Comment:
Yes. There's a "Create a new account" link on the Accounts index page (for admins)